chcemvediet/apps/obligees/tests/test_forms.py

```diff
@@ -16,7 +20,9 @@ class ObligeeFieldWithTextInputWidgetTest(ObligeesTestCaseMixin, TestCase):
     """
 
     class Form(forms.Form):
-        obligee = ObligeeField()
```

Comment:
Budeme potrebovat ObligeeFieldWithTextInputWidgetTest, nikde v kode nevyuzivame ObligeeField s TextInput. Ak nie, potom by som premenoval ObligeeFieldWithObligeeWidgetWidget na ObligeeFieldTest.

Reply:
@viliambalaz Ano, ak `ObligeeField` nikde nepouzivame s inym widgetom nez `ObligeeWidget`, tak potom jeho testy pouzitia s inymi widgetami nepotrebujeme. Stacia testy jeho pouzitia s `ObligeeWidget`. Analogicky pre ostatne field/widget kombinacie. Zrejme historicky sa to pouzivalo aj s `TextInput` widgetom niekedy.

Reply:
:ok: